test/integration/suites/node-attestation/03-test-node-attestation

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Test node attestation api
+jointoken=`docker-compose exec -u 1001 -T spire-server /opt/spire/conf/server/node-attestation -testStep jointoken`
+echo "Created Join Token" $jointoken
+
+svid1=`docker-compose exec -u 1001 -T spire-agent /opt/spire/conf/agent/node-attestation -testStep jointokenattest -tokenName $jointoken`
```

Comment:
can you add more steps to validate what happens when calling all those "agent specific" RPCs with a regular SVID?